src/controller.rs

```diff
@@ -835,3 +913,21 @@ pub async fn run(state: State) -> anyhow::Result<()> {
         .await;
     Ok(())
 }
+
+pub(crate) fn common_labels() -> BTreeMap<String, String> {
+    BTreeMap::from_iter(vec![
+        (
+            "app.kubernetes.io/name".to_string(),
+            "wasmcloud".to_string(),
+        ),
+        (
+            "app.kubernetes.io/managed-by".to_string(),
+            "wasmcloud-operator".to_string(),
+        ),
+    ])
+}
+fn common_labels_selector() -> String {
+    format!(
+        "app.kubernetes.io/name=wasmcloud,app.{WASMCLOUD_OPERATOR_MANAGED_BY_LABEL_REQUIREMENT}"
```

Comment:
I think this results in:
```
app.kubernetes.io/name=wasmcloud,app.app.kubernetes.io/managed-by=wasmcloud-operator
```
should it be:
```
app.kubernetes.io/name=wasmcloud,app.kubernetes.io/managed-by=wasmcloud-operator
```
or perhaps:
```
app.kubernetes.io/name=wasmcloud-app,app.kubernetes.io/managed-by=wasmcloud-operator
```

Reply:
Worse than that, I think the label set won't actually match anything to reconcile anyway. It should just be `app.kubernetes.io/managed-by=wasmcloud-operator` if it should be used for both services and pods (services have their own names). I'll fix that

Reply:
🤔 If this is a label that you might use in `wadm`, I may need to update our label validation code. Unsure if I allow `=` in there

Reply:
@brooksmtownsend this is a label selector and not a label. As long as `app.kubernetes.io/managed-by: wasmcloud-operator` is valid then what you have in wadm is fine.

Reply:
OH I see, selector not label. Yeah that's valid 👍🏻